tools/rosmaster/src/rosmaster/main.py

```diff
@@ -108,6 +112,14 @@ def rosmaster_main(argv=sys.argv, stdout=sys.stdout, env=os.environ):
         import socket
         socket.setdefaulttimeout(float(options.timeout))
 
+    if options.master_logger_level:
+        level = {'debug': logging.DEBUG, 'info': logging.INFO, 'warn': logging.WARN, 'error': logging.ERROR, 'fatal': logging.FATAL}
```

Comment:
Since the dict contains multiple elements I would suggest naming the variable `levels`.